stix2/sro.py

```diff
@@ -4,13 +4,14 @@
 
 from .base import _STIXBase
 from .common import ExternalReference, GranularMarking
+from .markings import MarkingsMixin
 from .properties import (BooleanProperty, IDProperty, IntegerProperty,
                          ListProperty, ReferenceProperty, StringProperty,
                          TimestampProperty, TypeProperty)
 from .utils import NOW
 
 
-class Relationship(_STIXBase):
+class Relationship(_STIXBase, MarkingsMixin):
```

Comment:
Make a `STIXRelationshipObject` class to prevent adding the `MarkingsMixin` to both classes.